woocommerce/woocommerce-ios
# Remove Cancel on card reader updates at 100%

Closes #5324 

## Description

Card reader updates hover at 100% when they are almost complete, displaying 100% in the progress view, for up to 15s. We've taken the decision to remove the Cancel button once 100% is displayed, to avoid a merchant ending an update right before it finishes.

## Testing
1. Simulate an optional update using the instructions here: PdfdoF-fr-p2
2. Go to `⚙️ > In-Person Payments > Manage Card Reader`
3. Connect to the card reader simulator
4. Start the update
5. Observe that the cancel button is removed as soon as the progress indicator gets to 100%
6. Redo the update, and confirm that before 100% the cancel button functions as expected.


Update release notes:
